incubator/hnc/docs/user-guide/concepts.md

```diff
@@ -315,6 +316,40 @@ whichever labels they like. However, HNC will overwrite any changes made to
 these labels, so other applications can trust these labels for policy
 application.
 
+<a name="basic-exceptions"/>
+
+### Exceptions and propagation control
+
+By default, HNC propagates _all_ objects of a [specified type](how-to.md#admin-resources) 
+from ancestor namespaces to descendant namespaces. However, sometimes this is 
+too restrictive, and you need to create ***exceptions*** to certain policies. For example:
+
+* A ResourceQuota was propagated to many children, but one child namespace now 
+has higher requirements than the rest. Rather than getting rid of the quota in 
+the parent namespace, or raising the limit for everyone, you can stop the 
+quota in the parent from being propagated to that ***one*** child namespace, 
+allowing you to replace it with another, more suitable quota.
+
+* A RoleBinding allows any user to create subnamespaces under one namespace, but 
+we don’t want to allow those users to create additional levels of hierarchy 
+underneath those subnamespaces. So you can stop the role binding from being 
+propagated to ***any*** child namespace.
```

Comment:
Sorry, only new _terms_ should be both bolded and italicized. Words that we just want to emphasize should only be italicized:

***New term***  - word that gets _emphasized_

`***New term***  - word that gets _emphasized_`

So "any" and "one" should just be italicized, whereas the first time we use the term "exceptions" it need to be both bolded and italicized.

Sorry for being unclear.

Reply:
(The idea behind this is that bolded words are kind of like shouting, so we only want to shout for new words that are ***really important!!!*** But some words just need a _gentle_ stress).